daniel-robinson.txt

```diff
@@ -0,0 +1,63 @@
+What is the difference between a section and a div? A section usually contains several divs.
+What is CSS used for? CSS is used to style the content displayed by HTML.
+How to we select an element? Example - every h2 on the page. An element selector is used to style HTML elements. Ex: h2 {text-align:center;}
+What is the difference between a class and an id? - Give me an example of when I might use each one.
+How do we select classes in CSS? To select classes in CSS put a period in front of the element selector. Ex: ".intro" to select <div class="intro">
+How do we select a p element with a single class of “human”? Use .human to select a p element with a single class of "human".
+What is a parent child selector? When would this be useful? The parent child selector "parent > child" is used to select all elements that under that parent.
+How do you select all links within a div with the class of sidebar? Use #sidebar to select all links withing a dive with the class of sidebar.
+What is a pseudo selector? A pseudo selector selects and element without calling the element verbatim. Instead it identifies the element by its position; :first-child for example.
+What do we use the change the spacing between lines? The line-height property is used to change hte spacing between lines.
+What do we use to change the spacing between letters? Use the property "letter-spacing: <some-attribute>".
+What do we use to to change everything to CAPITALS? lowercase? Capitalize? Use the property "transform-text: <some-attribute>".
+How do I add a 1px border around my div that is dotted and black? Use the following selector and property: div { border: dotted 1px black; }
+How do I select everything on the page? Select every element on a page using the universal element selector: "*".
+How do I write a comment in CSS? To write a comment in CSS, write "/*" at the beginning and "*/" at the end of the comment.
+How do I find out what file I am in, when I am using the command line? 
+Using the command line - how do I see a list of files/folders in my current folder? To see a contenct list of your current folder/directory, type in "ls" for list.
+How do I remove a file via the command line? Why do I have to be careful with this? To remove a file via the command line, type "rm" + <file-name>. Removing a file should be done with caution as there is absolutely no way to retrieve the file onces removed/deleted.
+Why should I use version control? Version control allows multiple developers and teams of developers to work on the same project (repository or repo) remotely and simultaneously; avoiding overlap and wasted time by forking and cloning seperate versions of the same repo.
+How often should I commit to github? A commit should be done at least twice a day; before lunch and at the end of the work day. To be extra safe, commit before you leave your workstation.
```

Comment:
committing vs pushing is different! commits are like saving locally. "commit early and commit often".